src/main/java/org/folio/circulation/domain/CheckInProcessRecords.java

```diff
@@ -23,6 +25,7 @@
   private final ServicePoint checkInServicePoint;
   private final Request highestPriorityFulfillableRequest;
   private final String loggedInUserId;
+  private final DateTime systemDateTime = new DateTime(DateTimeZone.UTC);
```

Comment:
Take a look at `org.folio.circulation.support.ClockManager#getDateTime`.

We usually use it for such cases. It can be also easily mocked with `ClockManager.setClock(Clock.fixed(...))`.

Reply:
@bohdan-suprun Yes, we should be replacing the creating of new date times in production code with the clock

Reply:
The clockmanager is not testable in this case.

Reply:
Done.

Reply:
Done.